cli_client/python/timesketch_cli_client/commands/analyze.py

```diff
@@ -101,8 +95,7 @@ def list_analyzers(ctx, output):
             'json','text' or 'tabular'
     """
     sketch = ctx.obj.sketch
-    if not output:
-        output = ctx.obj.output_format
+    output = ctx.obj.output_format
```

Comment:
When I call `timesketch --sketch 1 analyze list` I get the following exception:
```
Traceback (most recent call last):
  File "/usr/local/bin/timesketch", line 33, in <module>
    sys.exit(load_entry_point('timesketch-cli-client', 'console_scripts', 'timesketch')())
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/src/timesketch/cli_client/python/timesketch_cli_client/commands/analyze.py", line 98, in list_analyzers
    output = ctx.obj.output_format
  File "/usr/local/src/timesketch/cli_client/python/timesketch_cli_client/cli.py", line 125, in output_format
    elif self.config_assistant.get_config("output"):
  File "/usr/local/lib/python3.10/dist-packages/timesketch_api_client/config.py", line 115, in get_config
    return self._config[name]
KeyError: 'output'
```

Looks to me like there is no default output defined?

If I run `timesketch --output-format json --sketch 1 analyze list` everything works. I would expect a default output format instead of an exception though.

Reply:
done

```
timesketch --sketch 1 analyze list
Using default output format: tabular
Name	Display Name	Is Multi
ssh_sessionizer	SSH sessions	False
sessionizer	Time based sessions	False
```